k2/torch/csrc/fsa_class.h

```diff
@@ -124,6 +124,14 @@ struct FsaClass {
    */
   torch::IValue GetAttr(const std::string &name) /*const*/;
 
+  torch::Tensor GetTensorAttr(const std::string &name) const {
```

Comment:
I think we'd better not expose the implementation details(i.e. tensor, raggedtensor).

Reply:
Different from training, there are only one or two attributes per FSA during decoding. I am not sure whether it's a good way to use a dict to save the attributes and to use IValue to add one more indirection.